san/query.py

```diff
@@ -3,13 +3,19 @@
 
 
 def get_gql_query(idx, dataset, **kwargs):
-    query, slug = parse_dataset(dataset)
-    return getattr(
-        san.sanbase_graphql,
-        query,
-        lambda *args, **kwargs: not_found(query)
-    )(idx, slug, **kwargs)
-
+    if "/" in dataset:
```

Comment:
What's `dataset` here? It seems to me that it's a string. 
Also, what's the logic behind `if "/" in dataset`?

Reply:
The dataset is a string, containing "metric/slug", so if it doesn't have a "/" there, then there isn't a slug. The problem is, that one of the tests is failing atm and I'm trying to figure that one out